### PR TITLE
Epic 7 - Marko Tymofeiuk

### PR DESCRIPTION
**Pull request by markotymofeiuk**

### CODE REVIEW

**Comment by okohuch** [comment]
стоп

**Comment by izagor** [comment]
+ 12-19-2023

File: ai_12/marko_tymofeiuk/Epic 7/control.cpp [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+    srand(static_cast<unsigned int>(time(nullptr)));
+
+    float floatMatrix[10][10];
+    int intMatrix[10][10];
+
+    for (int i = 0; i < 10; i++)
+    {
+        for (int j = 0; j < 10; j++)
+        {
+            floatMatrix[i][j] = pow(i, 2) / 3.0 + pow(cos(pow((i + j), 3)), 2);
```
Comment (fm2lpnu) [inline ai_12/marko_tymofeiuk/Epic 7/control.cpp:263]: Task 1 (a,b) is not implemented according to requirements.
Matrix had to be created in separate function and returned from it

File: ai_12/marko_tymofeiuk/Epic 7/control.cpp [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+
+    for (int i = 0; i < 10; i++)
+    {
+        for (int j = 0; j < 10; j++)
+        {
+            intMatrix[i][j] = pow(i, 2) / 3 + pow(cos(pow((i + j), 3)), 2);
+        }
+    }
+
+    processMatrix(floatMatrix, number * 2.5);
```
Comment (fm2lpnu) [inline ai_12/marko_tymofeiuk/Epic 7/control.cpp:275]: Task 2 is not implemented according to requirements
Multipliers has to be entered, not hardcoded
The function should return matrix

**Review by fm2lpnu — APPROVED** [review]

**Pull request merged**